src/v1/custom_conversions/organization_index_metric_descriptors.apib

```diff
@@ -0,0 +1,125 @@
+### Retrieve Organization Metric Descriptors [GET /organizations/{org_id}/metric_descriptors]
+
+This endpoint returns information about the names and formats of custom metrics created for a single organization.
+
+These metric descriptors can be:
+* (AdStage) Custom Conversions, describing user-uploaded custom data.
+* Network Custom Conversions, describing conversion data imported from supported remote netwokrs (currently Facebook and LinkedIn).
+* Calculated Metrics, describing user-defined combinations of other metrics (not supported in the V1 reporting endpoint).
+
+Additionally, AdStage will automatically derive several metrics that are listed in this endpoint based on both AdStage and Network Custom Conversions:
+* Conversion Value
+* Cost per Conversion
+* Conversion Rate
+* Return on Ad Spend (ROAS)
+
++ Parameters
+
+    + org_id: 40 (string)
+
+        Which organization you want descriptors for.
+
++ Request
+
+    + Headers
+
+            Accept: application/json
```

Comment:
Authorization header should be here as well.

Reply:
This is actually a public endpoint!

Reply:
<img width="1048" alt="Screen Shot 2019-08-01 at 4 02 46 PM" src="https://user-images.githubusercontent.com/4065355/62332842-fec4e280-b475-11e9-83d4-1407a30e2c39.png">

Reply:
Are you saying the organization-specific metric descriptors list is a public endpoint? If so, that's a very serious issue that we need to fix ASAP.

Reply:
Oh no; that's the cross channel endpoint that's public.

Fuzzy search just picked the wrong file in my editor

Reply:
- [x] added header